src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+function changeMainImage(photoLink) {
+  const largePhoto = document.querySelector('#largeImg');
```

Comment:
you can initiate variable largePhoto before this function. No need to create it every time you change image

Reply:
Oh, you're right😅 Stupid me...